portugues/colaborando.md

```diff
@@ -27,4 +27,8 @@ O Git te parece ser util, mas como você ainda não vive no século XXI, você e
 
 
 
-Mas te bateu uma fome danada e você não tem nada na geladeira e nenhum tostão.
\ No newline at end of file
+Mas te bateu uma fome danada e você não tem nada na geladeira e nenhum tostão.
+
+Vende seu PC?
+[SIM](venda/sim.md)
+[NÃO](venda/nao.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.